src/applications/find-forms/components/SearchResult.jsx

```diff
@@ -1,8 +1,28 @@
+// Node modules.
 import React from 'react';
 import moment from 'moment';
-
+// Relative imports.
 import * as customPropTypes from '../prop-types';
 
+const onDownloadLink = (form) => (event) => {
+  // Derive if the download is a PDF.
+  const isPDF = form?.attributes?.url.toLowerCase().includes('.pdf');
+
+  // IE10+ : (has Blob, but not a[download] or URL)
+  if (navigator.msSaveBlob && isPDF) {
```

Comment:
I'm curious if this should be contained to IE, because Chrome and other browsers experience the same issue with PDFs hosted on non-www domains. I think we should get clarification - should PDFs hosted on domains such as vba.va.gov also download to the tray? I'm not sure if we'd be able to use JS to accomplish this anyway, or if we'd run into a CORS issue.

Reply:
Ya apparently what's happening is any downloads that aren't coming from the same domain block the `download` functionality from what I've been reading

Reply:
So the only way I can think of doing this is to basically have a proxy endpoint for all of these PDF links on our own domain, but that sounds 1) not very feasible and 2) absolutely ridiculous lol

Reply:
The JS solution would also fail for external domains, right?

Reply:
I'm fairly confident, though I don't know for certain yet